docs/sources/k6-studio/troubleshoot.md

```diff
@@ -28,5 +28,6 @@ The application logs are saved in the following directory:
 
 - On macOS: ~/Library/Logs/k6 Studio/k6-studio.log
 - On Windows: %USERPROFILE%\AppData\Roaming\k6 Studio\logs\k6-studio.log
+- On Linux:
```

Comment:
@Llandy3d do you know the path for Linux here as well?

Reply:
@cristianoventura do you know this one ? 🤔

Reply:
On Linux, the path is `~/.config/k6 Studio/logs/k6-studio.log`

Reply:
Thank you! 🙇